releases/v19.2.0-beta.20191014.md

```diff
@@ -0,0 +1,94 @@
+---
+title: What&#39;s New in v19.2.0-beta.20191014
+toc: true
+summary: Additions and changes in CockroachDB version v19.2.0-beta.20191014 since version v19.2.0-beta.20190930
+---
+
+## October 14, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-beta.20191014.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-beta.20191014.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-beta.20191014.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta.20191014
+~~~
+
+### SQL language changes
+
+- The new `SHOW RANGE ... FOR ROW` statement identifies the range containing a specific row of a table or secondary index. This is particularly useful for validating the location of a specific piece of data when using [partitions](../v19.2/topology-geo-partitioned-replicas.html) to control data location for improved performance or GDPR compliance. [#40954][#40954]
+- The [`SHOW PARTITIONS`](../v19.2/show-partitions.html) statement now displays inherited as well as explicitly set zone configurations for partitions. Also, it is now possible to retrieve full zone configurations via the `full_config_sql` and `full_config_yaml` columns in the `crdb_internal.zones` table, and the new `zone_id` and `subzone_id` columns in the `crdb_internal.partitions` table point to partitions' corresponding zone configurations in `crdb_internal.zones`. [#40493][#40493]
```

Comment:
We have deprecated the YAML config format since 19.1, so it is preferable not to refer to it here.

Also the proto format is definitiely a reserved feature and not to be advertised in docs (yet)